src/App.js

```diff
@@ -2,12 +2,66 @@ import React from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import GoodsList from './GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.PureComponent {
+  state = {
+    dataLoaded: false,
```

Comment:
do you really need this variable? just pass variable arrayOfGoods to render method. It will be empty if data is not loaded yet